app/helpers/calculators_helper.rb

```diff
@@ -0,0 +1,7 @@
+module CalculatorsHelper
+  def leave_pot_untouched_slider_max(contribution)
+    max = contribution * 5
+    max = 500 if max.zero?
+    max
```

Comment:
I was going to say the same, but agree with @benbarnett - I think this is clearer than a ternary.